packages/office-ui-fabric-react/src/components/Rating/Rating.Props.ts

```diff
@@ -53,6 +54,13 @@ export interface IRatingProps extends React.HTMLProps<HTMLElement> {
    * Optional id of label describing this instance of Rating
    */
   ariaLabelId?: string;
+
+  /**
+   * Optional setting to represent aggregate rating values on rating control.
+   * If set, rating control will be readonly rating control with partial star support.
+   * Also if set, this overrides the disabled prop
+   */
+  aggregate?: boolean;
```

Comment:
i can't find my earlier comment. I think this really needs to be re-evaluated with better props that make sense for the consumer.

For example; what does aggregate even mean in this context? Why would it become readonly? Why would it override disabled? 

A better approach might be:

```ts
/** Optional flag to make the component non interactive. **/
readOnly?: boolean; 
```

Then always support partial stars and let the caller decide if they want to pass in whole numbers or decimals.